frontend/src/index.tsx

```diff
@@ -35,8 +30,9 @@ ReactDOM.render(
           <Route path="/search" element={<Search />} />
           <Route path="/about" element={<About />} />
```

Comment:
Rather than creating an about path I would just redirect to the github repo when users click on "about us"